static/src/stylesheets/ie9.content.garnett.scss

```diff
@@ -3,4 +3,4 @@
 $browser-supports-flexbox: false;
 $browser-supports-columns: false;
 
-@import 'head.facia.pc';
+@import 'content.garnett';
```

Comment:
think git's got a little confused about what changed into what here, but essentially we still have ie9.content.garnett.scss loading `content.garnett`